packages/expo-font/plugin/src/withFontsAndroid.ts

```diff
@@ -26,3 +36,38 @@ export const withFontsAndroid: ConfigPlugin<string[]> = (config, fonts) => {
     },
   ]);
 };
+
+export const withXmlFontsAndroid: ConfigPlugin<XmlFonts[]> = (config, fonts) => {
+  return withMainApplication(config, async (config) => {
+    for (const { fontName, fontFiles } of fonts) {
+      const xmlFileName = fontName?.toLowerCase().replace(/ /g, '_')!;
```

Comment:
why do we not use `normalizeFilename` here also? We should be careful about special characters in the file name